crates/solvers/src/boundary/naive.rs

```diff
@@ -138,6 +138,7 @@ pub fn solve(
                 )
             })
             .collect(),
+        gas: None,
```

Comment:
Comment your consideration from the PR description in code here please so future maintainers have context on why this is `None`